app/jenkins-for-jira-ui/src/components/JenkinsSetup/JenkinsSetup.tsx

```diff
@@ -202,6 +241,14 @@ const IAmTheJenkinsAdmin = ({
 	);
 };
 
+const enum CopyButtonNameEnum {
```

Comment:
We want to avoid using enums in ts

More info [here](https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh#:~:text=Excessive%20use%20of%20regular%20ENUMs,scalability%20issues%2C%20and%20maintainability%20issues)

TLDR: they aren't typesafe, aren't as performant as objects or union/literal types and there are alternatives that are easier to work with

Could replace it with something like this:

const CopyButtonName = {
    AdminSiteName: 'adminSiteName',
    AdminWebhook: 'adminWebhook',
    AdminSecret: 'adminSecret',
    NonAdminWebhook: 'nonAdminWebhook',
    NonAdminSecret: 'nonAdminSecret'
} as const;
or

type CopyButtonName = 'adminSiteName' | 'adminWebhook' | 'adminSecret' | 'nonAdminWebhook';